Tools/SandboxTest.ps1

```diff
@@ -1,39 +1,66 @@
-# Parse Arguments
+# Parse arguments
 
 Param(
-  [Parameter(Mandatory, HelpMessage = "The path for the Manifest.")]
-  [String] $Manifest
+  [Parameter(Position=0, HelpMessage = "The Manifest to install in the Sandbox.")]
+  [String] $Manifest,
+  [Parameter(Position=1, HelpMessage = "The script to run in the Sandbox.")]
+  [ScriptBlock] $Script,
+  [Parameter(HelpMessage = "The folder to map in the Sandbox.")]
+  [String] $MapFolder = $pwd
 )
 
-if (-not (Test-Path -Path $Manifest -PathType Leaf)) {
-  throw 'The Manifest file does not exist.'
+$ErrorActionPreference = "Stop"
+
+$mapFolder = [System.IO.Path]::GetFullPath($MapFolder)

```

Comment:
I believe that GetFullPath will generate the path with the current directory where the PowerShell process started and will produce the wrong path if the user is running PowerShell, change directories and use the script.

I think it will be safer to use a combination of Get-Location + GetFullPath(String, String) with a prior call to IsPathFullyQualified to know if resolving the relative path is needed.

Reply:
It resolves the path in the situation you described. Since we're using `pwd`, it will always be right.

Reply:
The GIF in the PR description demonstrates that.

Reply:
NVM, it does not demonstrate that by this does:

![ZFpKotXBpO](https://user-images.githubusercontent.com/29582865/91506229-038ddd00-e8a8-11ea-979e-50e28532b3af.gif)

Reply:
Yes, if `pwd` is used it works as expected. The problems comes when the user specifies MapFolder

![image](https://user-images.githubusercontent.com/36937303/91507041-961e8400-e888-11ea-8148-9e4166e78437.png)

Reply:
You are totally right. 

I think `Resolve-Path` should be enough, don't you think?

Reply:
Thanks for raising this, I just noticed that installing a manifest provided would not work if the path for it wasn't relative to the map folder. I fixed that by the way.